src/Hosting/Hosting/src/WebHostBuilder.cs

```diff
@@ -212,7 +215,8 @@ IServiceProvider GetProviderFromFactory(IServiceCollection collection)
             }
         }
 
-        private IServiceCollection BuildCommonServices(out AggregateException hostingStartupErrors)
+        [MemberNotNull(nameof(_options))]
```

Comment:
weird

Reply:
It says that the field is non-null at the end of this method call.